client/src/components/CategoryShow.js

```diff
@@ -33,7 +33,8 @@ const CategoryShow = (props) => {
     pastasList = `There are currently no pastas added to this category`;
   } else {
     pastasList = category.pastas.map((pasta) => {
-      return <PastaTile key={pasta.id} {...pasta} />;
+      return <PastaTile 
+      key={pasta.id} {...pasta} />;
     });
```

Comment:
The previous version was formatted better on a single line with the return, otherwise break it up to separate lines
```js
return ( 
  <PastaTile 
      key={pasta.id} 
      {...pasta} 
    />;
)
```